src/styles.css

```diff
@@ -0,0 +1,206 @@
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.logo {
+  cursor: pointer;
+  display: block;
+  height: 40px;
+}
+
+.nav__item {
+  text-transform: uppercase;
+  white-space: nowrap;
+  align-items: center;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  text-decoration: none;
+  align-items: center;
+  height: 60px;
+  cursor: pointer;
+  color: var(--primary-text-color);
+  transition: color 0.3s;
+}
+
+.nav__link:hover {
+  color: var(--button-color);
+}
+
+.nav__link.is-active {
+  color: var(--button-color) ;
+}
+
+.nav__link.is-active::after {
+  background-color: var(--button-color);
+  border-radius: 2px;
+  height: 4px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  content: "";
+}
+/* #endregion */
+
+/* #region Card */
+.card {
+  box-sizing: border-box;
+  color: var(--primary-text-color);
+  padding: 32px 16px 16px;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__tittle {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  margin-top: 0;
+}
+
+.card__code {
+  margin: 0 0 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-text-color);
+}
+
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  font-weight: 400;
+  font-size: 10px;
+  margin-bottom: 24px;
+  line-height: 14px;
+}
+
+.card__price-container {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  line-height: 18px;
+}
+
+.card__price-tag {
+  font-size: 12px;
+  color: var(--secondary-text-color);
+  font-weight: 400;
+}
+
+.card__price-amount {
+  color: inherit;
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 12px 0;
+  background-color: var(--button-color);
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-transform: uppercase;
+  border: 1px solid var(--button-color);
+  border-radius: 5px;
+  color: #f3f3f3;
+}
+
+.card__button:hover {
+  background-color: #fff;
+  color: var(--button-color);
+}
+/* #endregion */
+
+/* #region Stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 > :nth-child(-n + 1),
+.stars--2 > :nth-child(-n + 2),
+.stars--3 > :nth-child(-n + 3),
+.stars--4 > :nth-child(-n + 4),
+.stars--5 > :nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
hmm, weren't you supposed to use `for` loop?
and i am suspicious - have you really checked vs the checklist?

Reply:
I did! I use the loop inside the scss file!  :D